dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpBaseMessageImpl.java

```diff
@@ -2941,10 +2943,12 @@ else if (getServiceContext().getHttpConfig().onlySameSiteStar()) {
             if (getServiceContext().getHttpConfig().useSameSiteConfig() && cookie.getAttribute("samesite") != null) {
                 boolean sameSiteNoneUsed = cookie.getAttribute("samesite").equalsIgnoreCase(HttpConfigConstants.SameSite.NONE.getName());
                 if(getServiceContext().getHttpConfig().getPartitioned() && sameSiteNoneUsed) {
-                    if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
-                        Tr.debug(tc, "[2] Setting the Partitioned attribute for SameSite=None");
+                    if(cookie.getAttribute("partitioned") == null) {  // null means no value has been set yet
+                        if (TraceComponent.isAnyTracingEnabled() && tc.isDebugEnabled()) {
+                            Tr.debug(tc, "[2] Setting the Partitioned attribute for SameSite=None");
+                        }
```

Comment:
Any need for beta checks in this area of code?

Reply:
same reasoning as in my other comment above -- getHttpConfig().getPartitioned() is beta guarded , so I don't think it needs it here.